apps/src/util/experiments.js

```diff
@@ -35,7 +35,7 @@ experiments.BYPASS_DIALOG_POPUP = 'bypass-dialog-popup';
 experiments.SPECIAL_TOPIC = 'special-topic';
 experiments.CLEARER_SIGN_UP_USER_TYPE = 'clearerSignUpUserType';
 experiments.OPT_IN_EMAIL_REG_PARTNER = 'optInEmailRegPartner';
-experiments.STUDIO_CERTIFICATE = 'studioCertificate';
+experiments.INSTRUCTOR_PREDICT_LEVEL_RESET = 'instructorPredictLevelReset';
```

Comment:
Merge conflict? I removed this in #48520

Reply:
Nice catch! yes, I resolved this manually and incorrectly